dashboards-observability/public/components/visualizations/charts/financial/gauge/gauge.tsx

```diff
@@ -41,17 +41,11 @@ export const Gauge = ({ visualizations, layout, config }: any) => {
 
   // data config parametrs
   const { dataConfig = {}, layoutConfig = {} } = userConfigs;
```

Comment:
Need the similar data config destructing here. If code changes are in other PR(s), then wait until that PR merged and then update this PR.